front/cypress/support/commands.ts

```diff
@@ -333,10 +338,10 @@ function apiUpdateCurrentUser(attrs: IUserUpdate) {
 function apiUpdateUserCustomFields(
   email: string,
   password: string,
-  custom_field_values: Record<string, any>
+  custom_field_values: Record<string, any>,
+  jwt?: any
```

Comment:
Do you ever pass it? TBH I'm not sure how this `jwt` change works including `_jwt: jwt`.

Is it possible to read `jwt` from cookies and then use by default in any request?